homeassistant/components/sensor/sma.py

```diff
@@ -209,4 +217,4 @@ def async_update_values(self):
     @property
     def unique_id(self):
         """Return a unique identifier for this sensor."""
-        return "sma-{}-{}".format(self._sensor.key, self._sensor.name)
```

Comment:
Will this change the unique ID of existing entities? Because that's a breaking change that we shouldn't do.

Reply:
It will change, but the original ID was not really unique - you can have multiple inverters. We can add an optional config option?

Reply:
No config option.

Bah, people are going to be confused, they will try to rename, it won't work as the old unique ID still occupies that entity ID but they can't change it without manually editing a file in `.storage`

Reply:
Is there any way to remove unique IDs? Or .pop() an existing one out of the registry?